src/pipeline/src/etl/transform/index.rs

```diff
@@ -13,20 +13,22 @@
 // limitations under the License.
 
 const INDEX_TIMESTAMP: &str = "timestamp";
+const INDEX_TIMEINDEX: &str = "timeindex";
```

Comment:
@killme2008 Would you prefer `index: timeindex` or just `index: time`?

Reply:
`index: time` looks good.

Reply:
Can we deprecate the `time` type too? Use `timestamp` type instead.